webapp/channels/src/components/update_user_group_modal/update_user_group_modal.tsx

```diff
@@ -31,10 +38,16 @@ export type Props = {
     };
 }
 
-const UpdateUserGroupModal = (props: Props) => {
+const UpdateUserGroupModal = ({
```

Comment:
Could you revert the non-button changes from this file and resubmit them separately as well? I know I've done some of that recently too, but I think we need to be better about not doing that for PRs that are already really involved.

It's especially important when it might have functionality changes like how you fixed the dependency array like with `updateNameState`. We don't want this PR to be blamed if fixing something like that actually causes something else to break 😅

Reply:
https://github.com/mattermost/mattermost/pull/28867